packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -1183,9 +1183,13 @@ export async function putOrganization(
 export async function getApiKeys(req: AuthRequest, res: Response) {
   const { org } = getOrgFromReq(req);
   const keys = await getAllApiKeysByOrganization(org.id);
+  const filteredKeys = keys.filter(
```

Comment:
Slightly confused on the UX here.
- Shouldn't an admin user be able to manage all keys?
- If so, should we also show which user owns each key?
- Is there any way to generate a key for a non-admin (as the UI doesn't allow non-admins to visit this page)?

Reply:
Regarding your first 2 questions, I think they're answered here: https://github.com/growthbook/growthbook/pull/1265#issuecomment-1548414344

Regarding the 3rd, not at this time, but they can generate a read-only token and share it with unprivileged users.